sig-cluster-lifecycle/annual-report-2022.md

```diff
@@ -4,64 +4,97 @@
+
+   - KEP-2915: Replace usage of the kubelet-config-x.y naming in kubeadm, using UnversionedKubeletConfigMap alpha:v1.23; beta: v1.24; GA: v1.25; Removed: v1.26. This is not included probably because it started in 2021.
+   - KEP-1739: kubeadm customization with patches: kubeadm: add support for patching a "kubeletconfiguration" target #110405 v1.25. This is similar that the KEP started before 2022, and the only kubeletconfiguration patching was supported in 2022.
+   - Some KEPs were retroactively updated to GA, since the work there was done without KEP updates.
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - [cluster-api-addon-provider-helm](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/blob/main/OWNERS)
+   - [etcdadm](https://github.com/kubernetes-sigs/etcdadm)
+   - [cluster-addons](https://github.com/kubernetes-sigs/cluster-addons)
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We started collecting metrics from all of our subprojects using an
+     [yearly survey](https://forms.gle/xZn8DXww4XxPsXvCA). It contains a number of questions
+     related to project health, OWNERS files, contributor onboarding, etc. What we saw for this annual
+     report is that our OWNERS files and SIG README.md are mostly up-to-date and that subprojects are doing
+     what they can to onboard new contributors. We are considering preparing some actions to better educate
+     our subprojects about onboarding contributors and graduating more OWNERS. We did not do a survey
+     for 2022.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - As per the 2021 survey, we are seeing a good number of projects to keep their contributing
+     documentation up-to-date. We are drafting action items for the leads to improve the understanding
+     around [CONTRIBUTING.md] management. We have a few projects that have participated in LFX and GSoC
+     (k0ps, cluster-addons, kubeadm).
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - Our subprojects have implemented their own contributing process and they have their own criteria
+     for contributors to climb the ladders. We have not seen complains from new or existing
+     contributors related to contributor guides. The SIG leads are open to help subproject leads with tips
+     on this topic.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, as confirmed by our survey.
```

Comment:
Is there a public link to the survey results?

The following dashboards can also be linked here:
- https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Cluster%20Lifecycle&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&viewPanel=1&from=1641016800000&to=1672466400000
- https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Cluster%20Lifecycle%20(Cluster%20API)&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&viewPanel=1&from=1641016800000&to=1672466400000